### PR TITLE
Incorrect label for SCIM

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Email request from @karanharibhakti-okta to correct the name of SCIM. No JIRA, minor fix.

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@karanharibhakti-okta This fix was trapped in my email queue. Can you approve if it's correct? Thanks!

File: _source/_use_cases/integrate_with_okta/provisioning.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ Related articles:
 
 * [Single Sign-On](/use_cases/integrate_with_okta/sso-with-saml)
 * [Promotion](/use_cases/integrate_with_okta/promotion)
-* [OIN FAQs](/use_cases/integrate_with_okta/oan-faqs)
\ No newline at end of file
+* [OIN FAQs](/use_cases/integrate_with_okta/oan-faqs)
```
Comment (mystiberry-okta) [inline _source/_use_cases/integrate_with_okta/provisioning.md]: No change for line 22, not sure why there's one indicated here.

Comment (bdemers) [reply _source/_use_cases/integrate_with_okta/provisioning.md]: i was just gong to ask that :)  extra new line at the end of the file?

Comment (bdemers) [reply _source/_use_cases/integrate_with_okta/provisioning.md]: it is whitespace issue, though i'm not sure which char, or why it isn't showing up in the difff

**Pull request merged**